stories/Templates/PublicationPage/PublicationPage.jsx

```diff
@@ -41,94 +41,128 @@ export const PublicationPage = ({
         languageswitcherData={languageswitcherData}
         locationData={locationData}
         langSelect={langSelect}
-      />
-      <div className="grid-container fluid publication-bottom">
-        <div className="grid-x grid-margin-x">
-          <div className="cell small-12 medium-12 large-11 large-offset-1 medium-offset-0">
-            <div className="column">
-              <Breadcrumbcomponent data={data} active={active} />
+      ></GlobalHeader>
+        <div className="grid-container fluid publication-bottom">
```

Comment:
@neelesh-tippe Change the class `publication-bottom` to `publication-page`.

Reply:
changes done